semanticcpg/src/test/scala/io/shiftleft/passes/MethodDecoratorPassTests.scala

```diff
@@ -20,7 +21,7 @@ class MethodDecoratorPassTests extends WordSpec with Matchers {
 
     method --- EdgeTypes.AST --> parameterIn
 
-    val methodDecorator = new MethodDecoratorPass(graph)
+    val methodDecorator = new MethodDecoratorPass(new Cpg(graph.graph))
```

Comment:
since we call this a lot, we could add the following to the Cpg companion object:
```
def apply(graph: ScalaGraph) = new Cpg(graph.graph)
```
and then replace this line (and many others) with
```
val methodDecorator = new MethodDecoratorPass(Cpg(graph))
```

Reply:
n.b. this pattern is used a lot in the CS repo as well, so just adjust the companion PR as well

Reply:
Good idea. I'll merge this one and then do that in a separate PR.